src/plugins/explore/public/components/tabs/action_bar/patterns_settings/patterns_settings_popover_button.tsx

```diff
@@ -3,26 +3,39 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { EuiButtonIcon, EuiPopover } from '@elastic/eui';
+import { EuiButtonEmpty, EuiPopover } from '@elastic/eui';
 import React, { useState } from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import { useSelector } from 'react-redux';
 import { PatternsSettingsPopoverContent } from './patterns_settings_popover_content';
+import { selectPatternsField } from '../../../../application/utils/state_management/selectors';
 
 export const PatternsSettingsPopoverButton = () => {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const patternsField = useSelector(selectPatternsField) ?? '';
 
   const openPopover = () => setIsPopoverOpen(true);
   const closePopover = () => setIsPopoverOpen(false);
 
   return (
     <EuiPopover
       button={
-        <EuiButtonIcon
+        <EuiButtonEmpty
           size="s"
           data-test-subj="patternsSettingButton"
           aria-labelledby="patternsSettingsButton"
-          iconType="gear"
+          iconType="arrowDown"
+          iconSide="right"
           onClick={openPopover}
-        />
+        >
+          <FormattedMessage
+            id="explore.discover.patterns.settings.fieldSelector"
+            defaultMessage="Patterns field: {patternsField}"
+            values={{
+              patternsField,
+            }}
+          />
+        </EuiButtonEmpty>
       }
       isOpen={isPopoverOpen}
       closePopover={closePopover}
```

Comment:
should we make the popover appear to the leftDown, since this appears so much on the right side of screen? See Popover prop `anchorPosition`: https://oui.opensearch.org/1.21/#/layout/popover

Reply:
actually the export button doesn't have it specified, when we reintroduce the export button if might feel weird to have one pop up left and one down. unless we primarily have it go down, which happens with `leftUp`. 

leftDown:
<img width="1812" height="791" alt="image" src="https://github.com/user-attachments/assets/ee74e9de-c838-48bf-a3ae-9a405e0b427c" />


leftUp:
<img width="1804" height="783" alt="image" src="https://github.com/user-attachments/assets/5c45306b-e92a-465d-a2ce-e9ef25cbdd33" />

Reply:
up to you, just try to make both experience consistent and looking good

Reply:
I've removed this change because it clashes with the supposed direction of the carrot. If there is a UI ask for it, I'll reinclude it, but for now, we're not going to have the export button and the patterns field selector appear at the same time, as export is not being included with patterns.